test/integration.test.js

```diff
@@ -1,142 +1,193 @@
 import test from 'ava';
 import pify from 'pify';
-import SemanticReleaseError from '@semantic-release/error';
 import releaseNotesGenerator from '../lib/index';
-import commits from './helpers/commits';
 
-const cwd = process.cwd();
+const url = 'https://github.com/owner/repo';
+const lastRelease = {gitHead: '1234'};
+const nextRelease = {gitHead: '456', version: '1.0.0'};
 
-test.afterEach.always(() => {
-  process.chdir(cwd);
-});
-
-test.serial('Use "conventional-changelog-angular" by default', async t => {
-  await commits(['fix(scope1): First fix', 'feat(scope2): Second feature']);
-  const changelog = await pify(releaseNotesGenerator)({});
+test('Use "conventional-changelog-angular" by default', async t => {
+  const commits = [
+    {hash: '111', message: 'fix(scope1): First fix'},
+    {hash: '222', message: 'feat(scope2): Second feature'},
+  ];
+  const changelog = await pify(releaseNotesGenerator)(
```

Comment:
can we use `require('util').promisify` instead of `pify`?

Reply:
Yes. But if we do that mean we won't be able to support node with `babel-register`.
Ref #485

Let's decide on that and I'll make the change (or not depending on the decision)

Reply:
I thought the babel-register was only for semantic-release-cli, not for semantic-release?

Reply:
That's the question. Do we want to use `babel-register` also for `semantic-release` to support node 4?
The subject was discusses in semantic-release/semantic-release#485

Reply:
No babel for normal semantic-release. I'm okay with LTS schedule, but only starting node 8